app/js/features/network/constants.js

```diff
@@ -7,3 +7,6 @@ export const UPDATE_BALANCE = 'NETWORK/UPDATE_BALANCE';
 export const UPDATE_BALANCE_SUCCEEDED = 'NETWORK/UPDATE_BALANCE/SUCCEEDED';
 export const UPDATE_BALANCE_FAILED = 'NETWORK/UPDATE_BALANCE/FAILED';
 
+export const GET_CONTACT_CODE = 'NETWORK/GET_CONTACT_CODE';
+export const GET_CONTACT_CODE_SUCCEEDED = 'NETWORK/GET_CONTACT_CODE_SUCCEEDED';
```

Comment:
what about: 
`NETWORK/GET_CONTACT_CODE/SUCCEEDED`
and
`NETWORK/GET_CONTACT_CODE/FAILED`
to stay consistent